### PR TITLE
Update tests for MSON refract

### PR DESCRIPTION
**Pull request by danielgtaylor**
This fixes the currently-failing tests in the `shared/refract` branch by updating the fixture to use MSON refract and commenting out some MSON-AST specific tests that will need to be updated in the future.

### CODE REVIEW

**Comment by pksunkara** [comment]
Tests failing. And are we not going to port the mocha tests?

**Comment by danielgtaylor** [comment]
@pksunkara test are (finally) passing and there are some basic MSON Refract tests included now. I removed some of the more complex tests because I don't think they belong in a binding, but rather in the upstream library (e.g. whether a mixin generates the right elements). What we really care about in the binding is that a very basic example works without errors and produces refract elements in the right place. Thoughts?

File: test/parser-mson-test.coffee [hunk 1]
```diff
@@ -1,151 +1,52 @@
...
     protagonist.parse source, (err, result) ->
       return done err if err
-      mson_ast = result.ast.content[0].content[0].actions[0].examples[0].responses[0].content[0]
-
-      # For debugging puposes:
-      #console.log JSON.stringify mson_ast, undefined, 2
-
+      attributes = result.ast.content[0].content[0].actions[0].examples[0].responses[0].content[0]
+      dataStructures = result.ast.content[1].content
+      console.log JSON.stringify(result.ast, null, 2)
```
Comment (pksunkara) [inline test/parser-mson-test.coffee]: Forgot to comment out?

**Comment by pksunkara** [comment]
One minor thing. Will merge it once it's fixed.

**Comment by danielgtaylor** [comment]
@pksunkara the extra `console.log` line has been removed :smile:

**Pull request merged**